features/cellular/framework/AT/AT_CellularStack.h

```diff
@@ -105,7 +105,8 @@ class AT_CellularStack : public NetworkStack, public AT_CellularBase {
             pending_bytes(0)
         {
         }
-        // Socket id from cellular device
+        // Socket identifier
```

Comment:
Should it still state the common case along with the special case? Common case being socket id is assigned by device/modem.